src/main/resources/interface-specification.yml

```
@@ -7360,6 +7360,7 @@ paths:
             items:
               $ref: '#/components/schemas/AttributeKind'
```

to comply with other configurations it is also necessary
```suggestion
            items:
              $ref: '#/components/schemas/AttributeKind'
            default: []
```